tests-integration/integration.py

```diff
@@ -20,11 +16,22 @@ def test_zim_main_page():
     """Main page specified, http://isago.rskg.org/, was a redirect to https
     Ensure main page is the redirected page"""
 
-    main_entry = get_zim_main_entry("/output/isago.zim")
+    main_entry = LibzimArchive("/output/isago.zim").main_entry
```

Comment:
Why using pylibzim's Archive for this?

Reply:
This is how it was done. I imagined there was good reasons for it. Should I understand there are none?

Reply:
Yes, probably dates from before we added scraperlib to zimit